tox.ini

```diff
@@ -2,29 +2,28 @@
 
 [gh-actions]
 python =
-    3.8: py38, apicheck, linkcheck
-    3.9: py39, flake8, pydocstyle, cov
+    3.9: py39, flake8, pydocstyle, cov, apicheck, linkcheck
     3.10: py310
     3.11: py311
     3.12: py312
+    3.13: py313
     pypy-3.10: pypy3
 
 [gh-actions:env]
 DJANGO =
-    3.2: django32
-    4.1: django41
     4.2: django42
     5.0: django50
     5.1: django51
+    5.2rc1: django52
```

Comment:
Would this fail if we had "5.2" instead of "5.2rc1"?

Reply:
Yes